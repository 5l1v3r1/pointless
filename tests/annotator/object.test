
b = "bar"
x = {
  foo = 123
  y = {
    z = foo
    c = b
  }
}