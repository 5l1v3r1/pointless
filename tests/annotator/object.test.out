
    z = foo
access: foo local: 2

    c = b
access: b global: 0

    z = foo
entry: z index: 0

    c = b
entry: c index: 1

    z = foo
entry: foo index: 2 upIndex: 0

  foo = 123
entry: foo index: 0

  y = {
entry: y index: 1

b = "bar"
entry: b index: 0

x = {
entry: x index: 1
